.evergreen/run-gssapi-tests.sh

```diff
@@ -9,10 +9,62 @@ cd ${PROJECT_DIRECTORY}
 source .evergreen/env.sh
 source .evergreen/cargo-test.sh
 
+# Source the drivers/atlas_connect secrets, where GSSAPI test values are held
+source "${DRIVERS_TOOLS}/.evergreen/secrets_handling/setup-secrets.sh" drivers/atlas_connect
```

Comment:
`drivers/atlas_connect` already had half of the secrets I needed, so I chose that vault and also added the remaining secrets I needed.